src/helpers/format.js

```diff
@@ -51,6 +52,22 @@ const toPeakStr = (peaks) => {
   return str;
 };
 
+let fixedWavelength = null;
+
+const extractFixedWavelength = (source) => {
+  const jcamp = Jcampconverter.convert(
+    source,
+    {
+      xy: true,
+      keepRecordsRegExp: /(CSAUTOMETADATA)/,
+    },
+  );
+  // eslint-disable-next-line prefer-destructuring
+  fixedWavelength = jcamp.info.$CSAUTOMETADATA.match(/FIXEDWAVELENGTH=([\d.]+)/)[1];
```

Comment:
- What will happen when `CSAUTOMETADATA` is not in the jcamp file?
- What will happen when `FIXEDWAVELENGTH` is not in the jcamp file?